contracts/utils/math/SafeCast.sol

```diff
@@ -218,7 +233,7 @@ library SafeCast {
      * - input must be less than or equal to maxInt256.
      */
     function toInt256(uint256 value) internal pure returns (int256) {
-        require(value < 2**255, "SafeCast: value doesn't fit in an int256");
+        require(value <= uint256(type(int256).max), "SafeCast: value doesn't fit in an int256");
```

Comment:
```suggestion
        // Note: Unsafe cast below is okay because `type(int256).max` is guaranteed to be positive
        require(value <= uint256(type(int256).max), "SafeCast: value doesn't fit in an int256");
```